Userland/Libraries/LibPDF/Fonts/Type1Font.cpp

```diff
@@ -104,6 +104,13 @@ u32 Type1Font::char_code_to_code_point(u16 char_code) const
     if (m_data.to_unicode)
         TODO();
 
+    if (m_data.encoding->is_windows() && char_code > 040 && !m_data.encoding->descriptors().contains(char_code)) {
+        // PDF Annex D table D.2, note 3:
+        // In WinAnsiEncoding, all unused codes greater than 40 (octal) map to the bullet character. However, only
+        // code 225 (octal) shall be specifically assigned to the bullet character; other codes are subject to future re-assignment.
+        return 8226; // Bullet.
+    }
+
```

Comment:
This should be applied to `TrueTypeFont::char_code_to_code_point()` as well.
Maybe the if-condition could be a helper in the Encoding class, something like `should_map_to_bullet(char_code)`